tests/Db/Syntax/ColumnDefinitionParserTest.php

```diff
@@ -4,24 +4,11 @@
 
 namespace Yiisoft\Db\Tests\Db\Syntax;
 
-use PHPUnit\Framework\TestCase;
-use Yiisoft\Db\Syntax\ColumnDefinitionParser;
-use Yiisoft\Db\Tests\Support\TestTrait;
+use Yiisoft\Db\Tests\AbstractColumnDefinitionParserTest;
 
 /**
  * @group db
  */
-final class ColumnDefinitionParserTest extends TestCase
+final class ColumnDefinitionParserTest extends AbstractColumnDefinitionParserTest
```

Comment:
Is it empty now? Does it make sense to have it like that?

Reply:
It extends `AbstractColumnDefinitionParserTest` and all the code for the test is there.
`Db` tests runs under `db` directory.